.gitignore

```diff
@@ -160,3 +160,6 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+# Ignoring data folder
+/data/
```

Comment:
We do not need to ignore the whole folder is better to leave it is was, with *.csv